post-til-private/src/main/java/no/difi/meldingsutveksling/ptp/MeldingsformidlerClient.java

```diff
@@ -79,13 +94,37 @@ public String getKeystorePassword() {
             return keystorePassword;
         }
 
+        public String getMpcId() {
+            return mpcId;
+        }
+
         public static Config from(DigitalPostInnbyggerConfig config, KeyStore keyStore) {
             final String url = config.getEndpoint();
             final String keystorePassword = config.getKeystore().getPassword();
             final String keystoreAlias = config.getKeystore().getPassword();
-            return new Config(url, keyStore, keystoreAlias, keystorePassword);
+            final String mpcId = config.getMpcId();
+            return new Config(url, keyStore, keystoreAlias, keystorePassword, mpcId);
         }
 
     }
 
-}
+    public class Kvittering {
+        private transient final ForretningsKvittering eksternKvittering;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the "transient" modifier from this field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS2065)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)